workflows/sock-shop-demo/usingCmdProbe/workflow.yaml

```diff
@@ -0,0 +1,670 @@
+  entrypoint: argowf-chaos
+  serviceAccountName: argo-chaos
+  securityContext:
+    runAsUser: 1000
+    runAsNonRoot: true
+  arguments:
+    parameters:
+      - name: adminModeNamespace
+        value: "litmus"
+  templates:
+    - name: argowf-chaos
+      steps:
+        - - name: install-application
+            template: install-application
+          - name: load-test
+            template: load-test
+        - - name: install-pod-delete
+            template: install-pod-delete
+          - name: install-pod-memory-hog
+            template: install-pod-memory-hog
+          - name: install-pod-cpu-hog
+            template: install-pod-cpu-hog 
+          - name: install-pod-network-loss
+            template: install-pod-network-loss
+        - - name: pod-memory-hog
+            template: pod-memory-hog
+          - name: pod-cpu-hog
+            template: pod-cpu-hog
+        - - name: pod-delete
+            template: pod-delete
+        - - name: pod-network-loss
+            template: pod-network-loss
+        - - name: revert-chaos
+            template: revert-chaos
+
+    - name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace=sock-shop","-typeName=resilient", "-timeout=400"] 
+
+    - name: install-pod-delete
+      inputs:
+        artifacts:
+          - name: install-pod-delete
+            path: /tmp/pod-delete.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                description:
+                  message: |
+                    Deletes a pod belonging to a deployment/statefulset/daemonset
+                kind: ChaosExperiment
+                metadata:
+                  name: pod-delete
+                spec:
+                  definition:
+                    scope: Namespaced
+                    permissions:
+                      - apiGroups:
+                          - ""
+                          - "apps"
+                          - "batch"
+                          - "litmuschaos.io"
+                        resources:
+                          - "deployments"
+                          - "jobs"
+                          - "pods"
+                          - "pods/log"
+                          - "events"
+                          - "configmaps"
+                          - "chaosengines"
+                          - "chaosexperiments"
+                          - "chaosresults"
+                        verbs:
+                          - "create"
+                          - "list"
+                          - "get"
+                          - "patch"
+                          - "update"
+                          - "delete"
+                      - apiGroups:
+                          - ""
+                        resources: 
+                          - "nodes"
+                        verbs:
+                          - "get"
+                          - "list"
```

Comment:
The permissions were updated in individual experiment CR can you check once and take it from there only for every experiment. like for [pod delete](https://github.com/litmuschaos/chaos-charts/blob/a5be9fb27c86421637f5ce34eb5da05460254f66/charts/generic/pod-delete/experiment.yaml#L17) deletecollection is also there in verbs.

Reply:
This we can do in a single PR for everyone.

Reply:
Ok sure @uditgaurav